test/utils/React-util.js

```diff
@@ -19,7 +19,7 @@ import {
 /**
  * Render a ReactElement and use helpful test utility methods against it.
  * @param {ReactElement} reactElement The element to render.
- * @returns {RenderedTree}
+ * @returns {RenderedTree} The return
```

Comment:
This is not needed anymore, I pushed an update to the lint rule to allow jsdoc returns without a description.  I pulled this branch, removed this, and ran `npm run lint` and it passes.  Hero me if this is not the case for you :)